catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/export/ExportItem.java

```diff
@@ -0,0 +1,67 @@
+package org.codice.ddf.commands.catalog.export;
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class ExportItem {
+    private String id = "";
+
+    private String metacardTag = "";
```

Comment:
With the only constructor forcing them to be set, technically no. But if any are ever added that don't change those, I'd prefer them to be the default string value instead of a null value because of how they are used in the export command.